cmd/node/config/p2p.toml

```diff
@@ -23,6 +23,10 @@
         [Node.Transports.TCP]
             ListenAddress = "/ip4/0.0.0.0/tcp/%d" # TCP listen address
             PreventPortReuse = false
+        [Node.ResourceLimiter]
```

Comment:
this section should be added on `fullArchiveP2P.toml` as well

Reply:
would have been caught at system test. Will add.

Reply:
done